### PR TITLE
Implemented back end. Set up routing to database from front end.

### PR DESCRIPTION
**Pull request by NatalieMonique111**
## Overview

_Replace with a general description or bullet-points of the change._

## Feedback Requested

_Tell your fellow devs what type of feedback you would like._

### CODE REVIEW

File: app/src/Region/index.js [hunk 1]
```diff
@@ -1,38 +1,94 @@
...
+    .map((row) => {
+      return {
+        id: idCnt++,
+        region: row.region,
+        site: row.site,
+        states: row.states, // state are string, despite holding multiple states sometimes.
+        unique_number: row.unique_number,
+      };
+    })
+    .filter((el) => el);
```
Comment (jenhuynh) [inline app/src/Region/index.js:35]: Nice use of filtering!

File: server/src/siteRouter.mjs [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import express from "express";
+
+import * as db from "./db.mjs";
+
+const siteRouter = express.Router();
+
+siteRouter.get("/", async (request, response) => {
```
Comment (jenhuynh) [inline server/src/siteRouter.mjs:7]: siteRouter looks good!

**Review by jenhuynh — APPROVED** [review]

File: app/src/Gallery/Gallery.js [hunk 3]
```diff
@@ -4,12 +4,13 @@ const Gallery = () => {
   const [testImg, setTestImg] = React.useState("");
 
   React.useEffect(() => {
-    const yourApiKey = "a79d815bc8a425b6e16b2436606de305";
+    const yourApiKey = "";
```
Comment (jenhuynh) [inline app/src/Gallery/Gallery.js:7]: Good job hiding the ApiKey for privacy/misuse by others!

**Comment by jenhuynh** [comment]
Overall good job, feel free to merge!

File: app/src/Gallery/Gallery.js (hunk 3 above)
Comment (tabbykatz) [reply app/src/Gallery/Gallery.js:7]: Did you get a new apiKey when you made this change?

File: app/src/Region/index.js [hunk 4]
```diff
@@ -1,38 +1,94 @@
 import React from "react";
```
Comment (tabbykatz) [inline app/src/Region/index.js:1]: https://twitter.com/dan_abramov/status/1308739731551858689

File: app/src/Gallery/Gallery.js (hunk 3 above)
Comment (NatalieMonique111) [reply app/src/Gallery/Gallery.js:7]: I was given two keys initially- I just disabled the one above.  I'm trying to do the following:  1) researching placing the API key as a GitHub secret...and also re-watching (2nd time) George's video on how to get the environment variable into Heroku.

**Review by tabbykatz — APPROVED** [review]

**Pull request merged**